net-misc/tigervnc/tigervnc-1.12.0-r1.ebuild

```diff
@@ -185,3 +193,14 @@ src_install() {
 		rm -r "${ED}"/usr/share/man/man8 || die
 	fi
 }
+
+pkg_postinst() {
+	if use server; then
+		elog "In order for Xsession to work properly, you also need"
```

Comment:
I wonder if we could use optfeature.eclass for this.

Reply:
Yes, and it works better.  It only displays the message if no display manager is installed.